homeassistant/components/rainmachine/__init__.py

```diff
@@ -222,12 +222,7 @@ async def async_setup_entry(  # noqa: C901
     ip_address = entry.data[CONF_IP_ADDRESS]
 
     try:
-        await client.load_local(
-            ip_address,
-            entry.data[CONF_PASSWORD],
-            port=entry.data[CONF_PORT],
-            use_ssl=entry.data.get(CONF_SSL, DEFAULT_SSL),
-        )
+        await load_rainmachine_client(client, ip_address, entry)
```

Comment:
Not sure if this reduced cognitive complexity! Prefer it as it was before. It was more readable and understandable.